### DISCUSSION TITLE
Bring back the RSS button

### DISCUSSION THREAD

**Discussion by sjehuda** [Floorp-Projects/Floorp#610] [Feature Request]
Browsers once had a built-in RSS button (￼) that was shown prominently in the browser's location bar to help users discover when a website had an RSS feed available. When navigating to any web page, it would auto-detect whether there was an RSS feed available, using a concept called auto-discovery. And clicking the button would simply to take you to the website's RSS feed.

Read the rest at https://openrss.org/blog/browsers-should-bring-back-the-rss-button

**Comment by creeper-0910** [comment]
duplicate https://github.com/Floorp-Projects/Floorp/discussions/570

**Comment by sjehuda** [comment]
On Sun, 03 Dec 2023 02:07:15 -0800
お餅のCreeeper ***@***.***> wrote:

> duplicate https://github.com/Floorp-Projects/Floorp/discussions/570
> 

Apologies.

That's my fault not being coordinated.

Thank you

**Reply by creeper-0910** [reply]
No problem👍
